docs/admin/Installation/Cloud-Providers.md

```diff
@@ -393,6 +392,9 @@ metadata:
 type: Opaque
 ```
 
+> **Note**
```

Comment:
Write it as 

:::note
.... the tex goes here. looks better
:::

Reply:
okay sure